test/acceptance/integration/tools/general_utils.py

```diff
@@ -291,7 +295,19 @@ def start_cb(parms):
     set_ssh_config(localhost)
     if world.cb_pid != '':
         stop_cb()
-    getattr(api, runner)('{bin_path} {parms} >> /tmp/cb_acceptance_test.log'.format(bin_path=config['bin_path'], parms=parms))
+    if config['log_path'] == '':
+        log_path = '/tmp/acceptance'
+    else:
+        log_path = config['log_path']
+    getattr(api, runner)('mkdir -p {log_path}'.format(log_path=log_path))
+    if config['pid_file'] == '':
+        pid_path = '/tmp/acceptance/contextBroker.pid'
```

Comment:
What would happend if two acceptance runs happens at same time? Maybe is safer to use mktemp for this (cleaning up the directory after that).

Same comment for log_path.